README.md

```diff
@@ -66,21 +68,23 @@ Sample data:
   ];
 ```
 
-#### 
- Head over to your dataService.js file, 'get' the quoteBook module, then add a property of service (or factory) to the app. It's important to remember the differences between a 'factory' and 'service' in Angular because they aren't written the same way. If you use angular.module('quoteBook').factory() make sure you create an object, add properties to it, then return that object. If you use angular.module('quoteBook').service(), you add properties to the 'this' keyword. More details here http://tylermcginnis.com/angularjs-factory-vs-service-vs-provider/
-* After you've created a new service or factory and made sure you included the file in your index.html file, add the Sample data above and place it in your service.
+####
+ Head over to your dataService.js file, 'get' the quoteBook module, create a service as a part of your 'quoteBook' app.
+* After you've created a new service or factory and made sure you included the file in your index.html file, add the Sample data above and place it in your service. Do not bind it to 'this'.
+
 
-Notice we didn't put our quotes array directly on 'this' or your object you're going to return. That's because we don't want this data to be directly accessed from outside of this service. Instead, we're going to create 'getter' and 'setter' methods in order to get, add to, or remove parts of the quotes array making the quotes array 'private' to this service.
 
 * Now that we have our data, let's set up ways to access that data.
-* Create three methods on your 'this' (service) or custom object (factory), one called getData, one called addData, and one called removeData
-* getData simply returns the quotes array
-* addData takes in a data object, verifies that data object has the proper keys (just text and author), then adds that object to the end of the quotes array
-* removeData takes in the text of a quote, loops through the quotes array, then removes the proper quote from the array.
+* Create three methods on 'this' service.
+  - getData - Returns the quotes array.
+  - addData - Takes a data object as an argument, verifies the data object has all the right values (text and author), pushes that object to the quotes array.
```

Comment:
"has all the right properties (text and author) and that they exist (aren't undefined)"